README.tfdoc.hcl

```
@@ -0,0 +1,338 @@
+
+      section {
+        title = "Module Configuration"
+
+        variable "module_enabled" {
+          type        = bool
+          default     = true
+          description = <<-END
+            Specifies whether resources in the module will be created.
+          END
+        }
+
+        variable "module_depends_on" {
+          type           = any
+          readme_type    = "list(dependencies)"
+          default        = []
+          description    = <<-END
+            A list of dependencies. Any object can be _assigned_ to this list to define a hidden external dependency.
+          END
+          readme_example = <<-END
+            module_depends_on = [
+              google_network.network
+            ]
+          END
+        }
+      }
+
+      section {
+        title = "Main Resource Configuration"
+
+        variable "service" {
+          required    = true
+          type        = string
+          description = <<-END
+            The service to enable.
+          END
+        }
+
+        variable "project" {
+          type        = string
+          description = <<-END
+            The project ID. If not provided, the provider project is used.
+          END
+        }
+
+        variable "disable_dependent_services" {
+          type        = bool
+          default     = false
+          description = <<-END
+            If `true`, services that are enabled and which depend on this service should also be disabled when this service is destroyed. If `false` or unset, an error will be generated if any enabled services depend on this service when destroying it.
+          END
+        }
+
+        variable "disable_on_destroy" {
+          type        = bool
+          default     = false
+          description = <<-END
+            If `true`, disable the service when the terraform resource is destroyed. May be useful in the event that a project is long-lived but the infrastructure running in that project changes frequently.
+          END
+        }
+
+        variable "timeouts" {
+          type        = map(string)
+          default     = {}
+          description = <<-END
+            How long certain operations are allowed to take before being considered to have failed.
+          END
+        }
+      }
+    }
+  }
+
+  section {
+    title   = "Module Attributes Reference"
+    content = <<-END
+      The following attributes are exported in the outputs of the module:
+
+      - **`module_enabled`**
+
+        Whether this module is enabled.
+
+      - **`service`**
+
+        All service api attributes.
+
+      <!-- all outputs in outputs.tf-->
+    END
+  }
+
+  section {
+    title = "External Documentation"
+
+    section {
+      title   = "Google Documentation"
+      content = <<-END
+        <!-- markdown-link-check-disable -->
+
+      - https://cloud.google.com/service-usage/docs/overview
+      END
+    }
+
+    section {
+      title   = "Terraform Google Provider Documentation:"
```

```suggestion
      title   = "Terraform Google Provider Documentation"
```

titles should never have `:` in best case ;)

---

Removed in latest commit.